src/CribblyBackend/Controllers/SampleController.cs

```diff
@@ -16,6 +17,14 @@ public async Task<IActionResult> Get()
                 return Ok("Hello, cribbly!");
             });
         }
+
+        [HttpGet("aboutme")]
+        [Authorize]
+        public IActionResult GetAboutMe()
+        {
+            return Ok(new { AuthId = User.GetAuthProviderId(), Email = User.GetEmail() });
+        }
```

Comment:
This doesn't really need to be here, but it's a nice demo endpoint to make sure your JWT contains what you think it does